modules/core/shared/src/main/scala/lucuma/core/syntax/BoundedInterval.scala

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+package lucuma.core.syntax
+
+import cats.Order
+import cats.syntax.all._
+import spire.math.Bounded
+import spire.math.Interval
+final class BoundedIntervalOps[A](val self: Bounded[A]) extends AnyVal {
+
+  /** True if this and the other interval abut each other. */
+  def abuts(other: Bounded[A])(implicit order: Order[A]): Boolean =
```

Comment:
`abuts`?

Reply:
This was present in our original `Interval`. I think I learned the meaning of "to abut" there.